source/management/configs.yml

```diff
@@ -150,6 +150,32 @@ paths:
         '429':
           $ref: '../shared/standard-responses.yml#/responses/rateLimitedResponse'
 
+  configs-id-services:
+    parameters:
+      - $ref: '../shared/parameters.yml#/id'
+    get:
+      summary: List services referenced by a config
+      description: |
+        Retrieves a list of service resources that reference a given config; supports filtering, sorting, and pagination. Requires admin access.
+      security:
+        - ztSession: [ ]
```

Comment:
Missing OIDC support

```
      security:
        - ztSession: []
        - oauth2: [openid]
```

Reply:
If this was copied from somewhere as a base, it is also missing it.

Reply:
went through and fixed all the missing oauth2 bits. Bumped the version